docs/pipeline-components-and-applications/iglu/igluctl-2/index.md

```diff
@@ -268,6 +269,31 @@ Example:
 }
 ```
 
+## static verify-parquet
+
+`igluctl static verify-parquet` verifies that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks). It reports the breaking schema versions.
+
+It accepts one required arguments:
+
+- `input` - path to your schema files.
+
+```bash
+$ ./igluctl static verify-parquet /path/to/static/registry
+```
+
+## static verify-redshift
+
+`igluctl static verify-redshift` verifies that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for loading into Redshift. It reports the schema criterions which belongs to breaking schema versions.
+
+It accepts two required arguments:
+
+- `server` - Iglu Server URL.
+- `apikey` - Iglu Server Read ApiKey
```

Comment:
Why does it take these args if it’s `static`?

Reply:
Seems like it was mistake. If you want, I can open a ticket to fix this. If we had to fix it, should we wait to update the docs until the fix ?

Reply:
Maybe we can just leave a note in the docs that “static” is a misnomer and will be fixed in the next version?